python-sdk/tutorials/automl-with-azureml/image-classification-multiclass/auto-ml-image-classification-multiclass.ipynb

```diff
@@ -441,8 +441,8 @@
     "}\n",
     "\n",
     "tuning_settings = {\n",
-    "    \"iterations\": 20,\n",
-    "    \"max_concurrent_iterations\": 4,\n",
+    "    \"iterations\": 10,\n",
```

Comment:
please adapt the comment as well: "We use Random Sampling to pick samples from this parameter space and try a total of 20 iterations with these different samples, running 4 iterations at a time on our compute target, which has been previously set up using 4 nodes. Please note that the more parameters the space has, the more iterations you need to find optimal models."